user_guide_src/source/installation/upgrade_validations.rst

```diff
@@ -0,0 +1,159 @@
+Upgrade Validations
+###################
+
+.. contents::
+    :local:
+    :depth: 1
+
+
+Documentations of Library
+=========================
+
+- `Form Validation Documentation Codeigniter 3.X <http://codeigniter.com/userguide3/libraries/form_validation.html>`_
+- :doc:`Validation Documentation Codeigniter 4.X </libraries/validation>`
+
+
+What has been changed
+=====================
+
+
+Upgrade Guide
+=============
+1. Within the view which contains the form you have to change:
+
+- ``<?php echo validation_errors(); ?>`` to ``<?= $validation->listErrors() ?>``
+- ``<?php echo form_open('form'); ?>`` to ``<?= form_open('form') ?>``
```

Comment:
`<?php echo form_open('form'); ?>` and `<?= form_open('form') ?>` are the same.
So you don't have to change.

Reply:
This is not a CI4 convention but pure PHP.